doc/report-issue.md

```diff
@@ -0,0 +1,37 @@
+## Report an issue
+
+This document outlines how to report an issue for the static analyzer and
+ensure that your bug report is complete with all required information.
+
+> [!TIP]
+> If you are a Datadog customer, send the bug report with all information
+> to your customer success manager.
+
+### What to send with your report
+
+When sending a bug report, make sure to include
+ - your `static-analysis.datadog.yml` file
+ - the output of the tool (e.g. CLI) when running either locally or in your CI/CD pipeline
+ - the SARIF file being produced (if any and available)
+ - the URL of your repository (even if private, it helps to troubleshoot)
+ - the exact command line used to run the analyzer
+
+### Performance issues
+
+If you are experiencing performance issues, enable the flag `--performance-statistics` when
+running the tool from the command line.
+
+When sending the report, make sure to include:
```

Comment:
is the reason we have this separate section because we don't require the SARIF file or exact command to troubleshoot performance issues?

Reply:
Because the command depends on their environment.